### DISCUSSION TITLE
Ruby ile makine öğrenmesi mümkün mü ?

### DISCUSSION THREAD

**Discussion by BunyaminAkdeniz** [19/classroom#55] [Anonim] [answered]
Hocam Merhabalar. Makine öğrenmesi örnekleri beni çok büyülüyor. Araştırdığımdaysa en yatkın dilin python olduğunu görüyorum. Makine öğrenmesini ruby ile yapmam mümkün mü yoksa gerçekten python bu konuda çok avantajlı ve pyhton öğrenmem mi gerekir? Ayrıca ruby ile makine öğrenmesi mümkünse, bana başlayabileceğim bir kaynakça önerebilir misiniz ? Teşekkür ederim

**Comment by roktas** [comment, marked as answer]
Soruna (pratik anlamda) net cevap vereyim: "hayır".  Python öğrenmelisin.  Ama sanıyorum Ruby'nin üzerine çok kolay olur. :-)

**Reply by BunyaminAkdeniz** [reply]
Teşekkür ederim hocam
